app/model/TelegramNotifier.php

```
@@ -0,0 +1,166 @@
+
+        $options = [
+            'http' => [
+                'method' => 'POST',
+                'header' => 'Content-type: application/x-www-form-urlencoded',
+                'content' => http_build_query($payload),
+                'timeout' => 5,
+            ],
+        ];
+        $context = stream_context_create($options);
+        @file_get_contents($url, false, $context);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 164. (undefined)

(ErrorControlOperator)